docs/userguide/odata_queries.rst

```diff
@@ -0,0 +1,23 @@
+OData Queries
+==============
+
+The REST service on the BIG-IP® device implements a subset of the Open Data Protocol, which allows a user to refine a set of data based on query parameters. This is especially useful when limiting the number of results returned on a ``get_collection()`` call. The way to use these query parameters with the f5-sdk is shown below:
+
+.. topic:: Examples
+
+**Filter example:** Retrieve only http profiles in a particular partition. Note this is an inclusive filter.
```

Comment:
All of the examples under ``.. topic::`` need to be indented (tab/4 spaces) or they won't build.